mmv1/products/vertexai/api.yaml

```diff
@@ -997,3 +999,208 @@ objects:
+        output: true
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Array
+        name: supportedOutputStorageFormats
+        description: 'Output only. The formats this Model supports in BatchPredictionJob.output_config. If both PredictSchemata.instance_schema_uri and PredictSchemata.prediction_schema_uri exist, the predictions are returned together with their instances. In other words, the prediction has the original instance data first, followed by the actual prediction content (as per the schema). The possible formats are: * `jsonl` The JSON Lines format, where each prediction is a single line. Uses GcsDestination. * `csv` The CSV format, where each prediction is a single comma-separated line. The first line in the file is the header, containing comma-separated field names. Uses GcsDestination. * `bigquery` Each prediction is a single row in a BigQuery table, uses BigQueryDestination . If this Model doesn''t support any of these formats it means it cannot be used with a BatchPredictionJob. However, if it has supported_deployment_resources_types, it could serve online predictions by using PredictionService.Predict or PredictionService.Explain. TODO(rsurowka): Analogous TODOs as for instances field above.'
+        output: true
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::String
+        name: createTime
+        description: Output only. Timestamp when this Model was uploaded into Vertex AI.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: updateTime
+        description: Output only. Timestamp when this Model was most recently updated.
+        output: true
+      - !ruby/object:Api::Type::Array
+        name: deployedModels
+        description: Output only. The pointers to DeployedModels created from this Model. Note that Model could have been deployed to Endpoints in different Locations.
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          name: deployedModel
+          description: Output only. The pointers to DeployedModels created from this Model. Note that Model could have been deployed to Endpoints in different Locations.
+          properties:
+              - !ruby/object:Api::Type::String
+                name: endpoint
+                description: Immutable. A resource name of an Endpoint.
+                input: true
+              - !ruby/object:Api::Type::String
+                name: deployedModelId
+                description: Immutable. An ID of a DeployedModel in the above Endpoint.
+                input: true
+      - !ruby/object:Api::Type::String
+        name: etag
```

Comment:
We can omit this field- it's used mechanically by the API, and shouldn't be relevant for end users.